model/cluster_utility_test.go

```diff
@@ -33,11 +35,11 @@ func TestGetUtilityVersion(t *testing.T) {
 		Fluentbit:          &HelmUtilityVersion{Chart: "6"},
 	}
 
-	assert.Equal(t, getUtilityVersion(u, PrometheusOperatorCanonicalName), &HelmUtilityVersion{Chart: "3"})
-	assert.Equal(t, getUtilityVersion(u, ThanosCanonicalName), &HelmUtilityVersion{Chart: "4"})
-	assert.Equal(t, getUtilityVersion(u, NginxCanonicalName), &HelmUtilityVersion{Chart: "5"})
-	assert.Equal(t, getUtilityVersion(u, FluentbitCanonicalName), &HelmUtilityVersion{Chart: "6"})
-	assert.Equal(t, getUtilityVersion(u, "anything else"), nil)
+	assert.Equal(t, &HelmUtilityVersion{Chart: "3"}, getUtilityVersion(u, PrometheusOperatorCanonicalName))
+	assert.Equal(t, &HelmUtilityVersion{Chart: "4"}, getUtilityVersion(u, ThanosCanonicalName))
+	assert.Equal(t, &HelmUtilityVersion{Chart: "5"}, getUtilityVersion(u, NginxCanonicalName))
+	assert.Equal(t, &HelmUtilityVersion{Chart: "6"}, getUtilityVersion(u, FluentbitCanonicalName))
+	assert.Equal(t, nilHuv, getUtilityVersion(u, "anything else"))
```

Comment:
nit: I think you can use `assert.Nil(...)` to avoid comparing to the nil value.
```suggestion
	assert.Nil(t, getUtilityVersion(u, "anything else"))
```